code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -102,3 +102,13 @@
 	if(istype(W, /obj/item/crowbar))
 		new /mob/living/simple_animal/pet/cat(loc)
 	return ..()
+
+/obj/structure/largecrate/secway
+	name = "secway crate"
+	icon_state = "lisacrate"
+
+/obj/structure/largecrate/secway/attackby(obj/item/W as obj, mob/user as mob, params)
```

Comment:
```suggestion
/obj/structure/largecrate/secway/attackby(obj/item/W, mob/user, params)
```
It came with the copy and paste but no need to specify it.

Reply:
Ah woops my mistake.